src/actions/fetch/copier.test.ts

```diff
@@ -75,7 +75,6 @@ describe('fetch:copier', () => {
 
   const mockReader: UrlReader = {
     readUrl: jest.fn(),
-    read: jest.fn(),
```

Comment:
Why is this deleted?

Reply:
it's no longer a parameter for UrlReader and isn't used